### ISSUE TITLE
Focus outline is clipped on linked widgets

### ISSUE THREAD

**Issue by cassidyjames** [elementary/stylesheet#904]
It's not the biggest deal, but a small detail I noticed; linked widgets' outlines appear to be clipped or just not implemented evenly on the linked sides.

** Combo boxes**

![Screenshot from 2020-12-14 15-26-17](https://user-images.githubusercontent.com/611168/102143227-dfd3b000-3e20-11eb-9b7f-088736467a73.png)

**Stack switcher in header bar**

![Screenshot from 2020-12-14 15-27-38](https://user-images.githubusercontent.com/611168/102143271-f37f1680-3e20-11eb-91d7-e88bc8462aaf.png)

**Comment by danrabbit** [comment]
Yeah it looks like there's some kind of issue with Z ordering. Adwaita appears to get around this so I'll have to look into how
